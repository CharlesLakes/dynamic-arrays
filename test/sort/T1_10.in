10
92 19 18 50 7 25 55 30 35 66