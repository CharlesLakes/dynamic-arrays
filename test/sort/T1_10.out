7 18 19 25 30 35 50 55 66 92